make.xsh

```diff
@@ -133,30 +133,56 @@ def start_backend_prod():
     __start_backend(env_file=PROD_ENV_FILE)
 
 @makefile.target(tag="run services locally")
-def backup_database_dev():
+def run_local_db():
     """
     Use the "database-backup" service in the "docker-compose.yml" file to backup
     all of the tables in the instace of Postgres running locally.
     """
     export_dot_env_vars(env_file=DEV_ENV_FILE)
+    export_rootski_profile_aws_creds()
+    $POSTGRES_HOST = get_localhost()
+    docker-compose run -d -p $POSTGRES_PORT:$POSTGRES_PORT postgres
+
+@makefile.target(tag="run services locally")
+def backup_local_db():
+    """
+    Use the "database-backup" service in the "docker-compose.yml" file to backup
+    all of the tables in the instace of Postgres running locally. You must have a
+    postgres container running which you can run using "make run-local-db"
+    """
+    export_dot_env_vars(env_file=DEV_ENV_FILE)
+    export_rootski_profile_aws_creds()
     $POSTGRES_HOST = get_localhost()
-    docker network prune --force
+    # Runs a postgres instance if one is not already running
+    docker ps | grep postgres || docker-compose run -d -p $POSTGRES_PORT:$POSTGRES_PORT postgres
     docker-compose run database-backup backup
 
 @makefile.target(tag="run services locally")
-def restore_database():
+def restore_local_db():
```

Comment:
Can you rename this to `restore_database` and verify that it works after running the new `make start-database` target?